netbox/netbox/authentication.py

```diff
@@ -47,6 +48,8 @@
     'salesforce-oauth2': ('Salesforce', 'salesforce'),
 }
 
+BASE_MICROSOFT_GRAPH_URL = 'https://graph.microsoft.com/v1.0/'
```

Comment:
We should probably move this inside `azuread_map_groups()` as its only used there.